docs/source/basics/parameter_types.md

```diff
@@ -0,0 +1,227 @@
+Parameter Types
+---
+
+The Latch SDK supports first-class static typing. All input parameters and
+output values must be annotated with types in the function signature. 
+
+These type annotations validate task values and guide construction of
+parameter-specific frontend components when your interface is generated at
+registration.
+
+## Supported Types
+
+Below is an exhaustive list of supported types currently shipped with the SDK.
+
+
+### Integers
+
+```
+@task
+def foo(
+  a: int = 10
+):
+  ...
+```
+
+### Floats
+
+```
+@task
+def foo(
+  a: float = 10.0
+):
+  ...
+```
+
+### Strings
+
+```
+@task
+def foo(
+  a: str = "bio"
+):
+  ...
+```
+
+### Files
+
+```
+@task
+def foo(
+  a: LatchFile = LatchFile("/root/data.txt")
```

Comment:
this seems inconsistent with the customizing_interfaces section cuz you use FlyteFiles/FlyteDirectories there instead of LatchFiles/LatchDirectories

Reply:
resolved